handlers/handlers.go

```diff
@@ -36,10 +37,11 @@ func Release(cfg config.Config, rc RenderClient, api ReleaseCalendarAPI) http.Ha
 	})
 }
 
-func release(w http.ResponseWriter, req *http.Request, userAccessToken, collectionID, lang string, rc RenderClient, api ReleaseCalendarAPI, _ config.Config) {
+func release(w http.ResponseWriter, req *http.Request, userAccessToken, collectionID, lang string, rc RenderClient, api ReleaseCalendarAPI, cfg config.Config) {
 	ctx := req.Context()
 
-	release, err := api.GetLegacyRelease(ctx, userAccessToken, collectionID, lang, req.URL.EscapedPath())
+	_, uri, _ := strings.Cut(req.URL.EscapedPath(), cfg.PrivateRoutingPrefix)
```

Comment:
trim prefix, in place of cut ?

Reply:
no I explicitly want to remove the prefix, so I have the correct path for retrieval from zebedee/relcal-api

Reply:
I beg your pardon :-) Yes indeed strings.TrimPrefix() is perfectly fine and better suited